pkg/chunk/purger/purger.go

```diff
@@ -24,6 +25,42 @@ import (
 
 const millisecondPerDay = int64(24 * time.Hour / time.Millisecond)
 
+type purgerMetrics struct {
+	deleteRequestsProcessedTotal      *prometheus.CounterVec
+	deleteRequestsChunksSelectedTotal *prometheus.CounterVec
+	deleteRequestsProcessingFailures  *prometheus.GaugeVec
+}
+
+func newPurgerMetrics(r prometheus.Registerer) *purgerMetrics {
+	m := purgerMetrics{}
+
+	m.deleteRequestsProcessedTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "purger_delete_requests_processed_total",
+		Help:      "Number of delete requests processed per user",
+	}, []string{"user"})
+	m.deleteRequestsChunksSelectedTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "purger_delete_requests_chunks_selected_total",
+		Help:      "Number of chunks selected while building delete plans per user",
+	}, []string{"user"})
+	m.deleteRequestsProcessingFailures = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "purger_delete_requests_processing_failure",
```

Comment:
What do you think to add an `s` to `failure`?
```suggestion
		Name:      "purger_delete_requests_processing_failures",
```